src/inc/quic_platform.h

```diff
@@ -323,6 +323,29 @@ CxPlatListPopEntry(
 #include "quic_hashtable.h"
 #include "quic_toeplitz.h"
 
+#ifdef DEBUG
+void
+CxPlatSetAllocFailDenominator(
+    _In_ int32_t Value
+    );
+
+int32_t
+CxPlatGetAllocFailDenominator(
+    );
+#endif
+
+inline
+BOOLEAN
+CxPlatIsRandomMemoryFailureEnabled(
+    )
+{
+#ifdef DEBUG
+    return CxPlatGetAllocFailDenominator() != 0;
+#else
+    return FALSE;
+#endif
+}
```

Comment:
Just define `CxPlatIsRandomMemoryFailureEnabled()` as `FALSE` if not debug mode instead.

Reply:
The compiler does some constants optimization, so either is functionally the same. But `#define CxPlatisRandomMemoryFailureEnabled() FALSE` makes life easier for the optimizer.